src/plugins/controls/public/time_slider/components/time_slider.tsx

```diff
@@ -41,12 +40,6 @@ export const TimeSlider: FC<Props> = (props: Props) => {
     return state.componentState.isOpen;
   });
 
-  const rangeRef = useRef<EuiDualRangeRef>(null);
-
-  const onPanelResize = (width: number) => {
-    rangeRef.current?.onResize(width);
-  };
```

Comment:
This ref and method call were removed because **EuiDualRange** now contains its own resize observer and automatically adjusts its width accordingly.